sdks/ruby/src/.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+AllCops:
+  TargetRubyVersion: 3.1
+
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
+
+Style/StringLiteralsInInterpolation:
```

Comment:
there are probably some rubocop recommendations for which rules to enable somewhere which might be useful here